test/MimeMapping.Tests/MimeMapping.Tests.csproj

```diff
@@ -2,19 +2,19 @@
   <PropertyGroup>
     <TargetFrameworks>net8.0;net6.0;net48</TargetFrameworks>
      <TargetFrameworks Condition="$([MSBuild]::IsOSUnixLike())">net8.0;net6.0</TargetFrameworks>
-    <RunSettingsFilePath>$(MSBuildThisFileDirectory)/test.runsettings</RunSettingsFilePath>
+    <RunSettingsFilePath>$(MSBuildThisFileDirectory)..\test.runsettings</RunSettingsFilePath>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
   
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
-    <PackageReference Include="MSTest.TestAdapter" Version="3.2.2" />
-    <PackageReference Include="MSTest.TestFramework" Version="3.2.2" />
+    <PackageReference Include="MSTest.TestAdapter" Version="3.3.1" />
+    <PackageReference Include="MSTest.TestFramework" Version="3.3.1" />
```

Comment:
same here

Reply:
Manual fix as in actual master https://github.com/zone117x/MimeMapping/blob/master/Test/Test.csproj